### DISCUSSION TITLE
[Enhancement] [config-provider] can message suppot more options?

### DISCUSSION THREAD

**Discussion by zengkaii** [element-plus/element-plus#18028] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `config-provider`

## Description

now [Config Provider message] only suppot [max] , hope suppot more options like [grouping], thanks.

### Additional comments

_(empty)_

**Comment by btea** [comment]
Can you explain in more detail?

**Reply by zengkaii** [reply]
> Can you explain in more detail?

now elmessage is suppot many attributes, like [grouping]....
If possible，have considered joining config-provider？thanks

**Reply by btea** [reply]
It sounds great to me. However, you can set a unified configuration by encapsulating a message method yourself.

**Reply by btea** [reply]
I created a PR #18106, but the process may not be that fast.

**Reply by zengkaii** [reply]
> It sounds great to me. However, you can set a unified configuration by encapsulating a message method yourself.

ok,thank
